src/com/sun/ts/lib/harness/keyword.properties

```diff
@@ -68,52 +68,372 @@ com/sun/ts/tests/connector/resourceDefs/ejb = javaee connector_resourcedef_ejb_o
 com/sun/ts/tests/concurrency = concurrency javaee
 
 com/sun/ts/tests/ejb = ejb javaee
-com/sun/ts/tests/ejb/ee/bb/entity/bmp = ejb_1x_optional javaee_optional
-com/sun/ts/tests/ejb/ee/bb/entity/cmp = ejb_1x_optional javaee_optional
-com/sun/ts/tests/ejb/ee/bb/entity/cmp20 = ejb_2x_optional javaee_optional
-com/sun/ts/tests/ejb/ee/bb/entity/lrapitest = ejb_1x_optional javaee_optional
+
+com/sun/ts/tests/ejb/ee/bb = javaee_optional ejb_1x_optional javaee_web_profile_optional
 
 com/sun/ts/tests/ejb/ee/bb/localaccess = ejb_1x_optional javaee_optional
-com/sun/ts/tests/ejb/ee/bb/localaccess/ebaccesstest = javaee_optional ejb_1x_optional javaee_web_profile_optional
-com/sun/ts/tests/ejb/ee/bb/localaccess/mdbqaccesstest = javaee_optional ejb_mdb_optional javaee_web_profile_optional
+com/sun/ts/tests/ejb/ee/bb/localaccess/ebaccesstest = javaee_optional ejb_1x_optional ejb_2x_optional javaee_web_profile_optional
```

Comment:
Is com/sun/ts/tests/ejb/ee/bb/localaccess/ebaccesstest marked as ejb_2x_optional because its using PortableRemoteObject indirectly?